test/capture.js

```diff
@@ -1,3 +1,21 @@
+define([ 'intern!tdd',
+        'intern/chai!expect',
+        '../pointerevents',
+        'src/boot',
```

Comment:
`src/PointerEvent` is incorrect, that's only the constructor. You want `pointerevents` from the root of the repo.